scripts/bootstrap_tdx.sh

```
@@ -1,296 +1,9 @@
-        echo "Set default boot entry using grub-set-default"
-    fi
-
-    if command -v grub-reboot >/dev/null 2>&1; then
-        grub-reboot 0
-        echo "Set next boot entry using grub-reboot"
-    fi
-
-    echo "GRUB configuration completed successfully for kernel ${new_kernel}"
-    return 0
+source_common() {
+    local script_dir=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)